src/EventListener/MergeUserWishlistItemsListener.php

```diff
@@ -70,13 +70,13 @@ private function resolveWishlist(Request $request, ShopUserInterface $shopUser):
 
         $userWishlist = $this->wishlistRepository->findByShopUser($shopUser);
 
-        if (null !== $cookieWishlist && null !== $userWishlist) {
+        if (null !== $userWishlist) {
             foreach ($cookieWishlist->getWishlistProducts() as $wishlistProduct) {
                 $userWishlist->addWishlistProduct($wishlistProduct);
             }
         }
 
-        if (null !== $cookieWishlist && null === $userWishlist) {
+        if (null === $userWishlist) {
```

Comment:
will it work outside of the API?

Reply:
https://github.com/TymekB ? 

@patrick477 on line 67 there is a check for $cookieWishlist